### PR TITLE
Fix CONTRIBUTING and README disparity

### LINKED ISSUE #67: Instructions in README and CONTRIBUTING are not aligned
All instructions about releasing a version should be in CONTRIBUTING file, not distributed accross two files:

https://github.com/Starcounter/StarcounterClientFiles/blob/3.x/README.md
https://github.com/Starcounter/StarcounterClientFiles/blob/3.x/CONTRIBUTING.md

As found in https://github.com/Starcounter/StarcounterClientFiles/pull/64#issuecomment-381175880

### PR DESCRIPTION
**Pull request by alshakero**
Fix #67

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
+## Testing your PR in TeamCity
+
+For your PR to be honoured by TeamCity, its name needs to start with `3.x` if you plan to merge it to `3.x` and with `1.x` if you plan to merge it to `1.x`. (eg: `3.x/fix-something`).
```
Comment (warpech) [inline CONTRIBUTING.md]: You read my mind, thanks for this change! Actually I would put the information about `1.x` in the README.md file of the `1.x` branch.

Pls change "For your PR" to "For your feature branch".

**Comment by tomalec** [comment]
Could we also improve
> 7. Still in the feature branch, **release new version** (3.x) of StarcounterClientFiles on GitHub. Remember to write the release notes.

The entire section is about releasing the section, which makes me fall into infinitive loop ;)

Does it mean
> 7. Still in the feature branch, push a tag (3.x.y), and add release notes to GitHub release of StarcounterClientFiles.

**Review by warpech — APPROVED** [review]
Lgtm

**Comment by tomalec** [comment]
As we discovered at https://github.com/Starcounter/KitchenSink/pull/281#issuecomment-383570423 my latest release I did according to this instructions went wrong.

Is there something missing in the descriptions, in the process itself, or have I simply made it incorrect?

**Comment by alshakero** [comment]
I think this was the `nuget` issue that you've already fixed?

**Edit: not really. I'll check the other issue**

**Comment by tomalec** [comment]
The reason of the problem was solved at 
https://github.com/Starcounter/CI.BuildSystem/commit/c53dc270651f826c7cd5d8ca7897f901d64a24fd and 
https://github.com/Starcounter/level1/commit/def3b8f8036ed8c74305aa700e14611a6efd2539#diff-2107037cf23c2ff91d0207fc9d099617

**Review by tomalec — APPROVED** [review]

**Pull request merged**